sda/simulation_test.go

```diff
@@ -43,7 +47,12 @@ func TestSimulationBF(t *testing.T) {
 	}
 }
 
-func TestBigTree(t *testing.T) {
+func TestSimulationBigTree(t *testing.T) {
+	t.Skip()
```

Comment:
You _really_ don't want this test, do you? I think keeping the `t.Skip()` in the `if` is enough...